docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-unified-data-model/conversions/index.md

```diff
@@ -157,6 +157,24 @@ Using our [Snowplow e-commerce tracking](/docs/collecting-data/collecting-from-o
     }
 ```
 
+</TabItem>
+<TabItem value="redshift" label="Redshift">
+
+```yml
+  snowplow__entities_or_sdes: [
+              {'schema': 'com_snowplowanalytics_snowplow_ecommerce_transaction_1', 'prefix': 'trans_entity', 'alias': 'tr', 'single_entity': true},
+              {'schema': 'com_snowplowanalytics_snowplow_ecommerce_snowplow_ecommerce_action_1', 'prefix': 'trans_event', 'alias': 'trev', 'single_entity': true}
+              ]
+
+      snowplow__conversion_events: [
+                  {
+          "name": "transact", 
+          "condition": "event_name = 'snowplow_ecommerce_action' and trans_event_type = 'transaction'",
+          "value": "trans_entity_revenue"
+          }
```

Comment:
The indent on these is off, and in general doesn't quite match the other examples. We should be consistent across them all I think (having the var names is better)

Reply:
Rewrote them as even I found them confusing without the wheres and whats, now it is copypastable.